ejb/src/test/java/biz/karms/sinkit/ejb/impl/CoreServiceEJBTest.java

```diff
@@ -44,8 +48,64 @@
     @InjectMocks
     private CoreServiceEJB coreService;
 
+/**
+ * Blacklist tests
+ */
+    @Test
+    public void processNotOnWhitelistActiveTest() throws Exception{
+
+        IoCRecord ioc= getIoCRecord(null, "facebook.malware.com","SomeFeed", 1, "malware");
+        // IoC is newer than oldest we allow
+        coreService.setIocActiveHours(24);
+        when(whitelistCacheService.get("malware.com")).thenReturn(null);
+        when(whitelistCacheService.get("facebook.malware.com")).thenReturn(null);
+        coreService.processIoCRecord(ioc);
+        verify(whitelistCacheService).get("malware.com");
+        verify(whitelistCacheService).get("facebook.malware.com");
+        assertEquals(null,ioc.getWhitelistName());
+        verify(archiveService).archiveReceivedIoCRecord(ioc);
+        verify(blacklistCacheService).addToCache(ioc);
+    }
+
+
     @Test
-    public void processExistingCompletedNoUpdateTest() throws Exception {
+    public void processOnWhitelistActiveTest() throws Exception{
+
+        Calendar expiresAt = Calendar.getInstance();
+        expiresAt.add(Calendar.DAY_OF_MONTH, 1);
+        IoCRecord ioc= getIoCRecord(null, "notmalware.webhosting.com",
+                "SomeFeed", 1, "malware");
+        // IoC is newer than oldest we allow
+
+        coreService.setIocActiveHours(24);
+        when(whitelistCacheService.get("webhosting.com")).thenReturn(null);
+        WhitelistedRecord white = createWhite("notmalware.webhosting.com" ,
+                "mywhitelist", expiresAt, true);
+        when(whitelistCacheService.get("notmalware.webhosting.com")).thenReturn(white);
+        coreService.processIoCRecord(ioc);
+        verify(whitelistCacheService).get("webhosting.com");
+        verify(whitelistCacheService).get("notmalware.webhosting.com");
+        assertEquals("mywhitelist",ioc.getWhitelistName());
+        verify(archiveService).archiveReceivedIoCRecord(ioc);
+        verify(blacklistCacheService).addToCache(ioc);
+
+    }
+
+    @Test (expected = TooOldIoCException.class)
+    public void processNotActiveTest() throws Exception{
+
+        IoCRecord ioc= getIoCRecord(null, "facebook.malware.com","SomeFeed", 52, "malware");
+        // IoC is newer than oldest we allow
+        coreService.setIocActiveHours(24);
+        coreService.processIoCRecord(ioc);
+    }
+
+    /**
+     *Whitelist tests
+     */
+
```

Comment:
@hugeox Arbitrary newlines, both in method bodies, between doc, etc. I don't mind having a bit different style, but stick to one and keep it, please.